src/coreclr/src/tools/crossgen2/ILCompiler.ReadyToRun/ILCompiler.ReadyToRun.csproj

```diff
@@ -3,6 +3,7 @@
     <OutputType>Library</OutputType>
     <AssemblyName>ILCompiler.ReadyToRun</AssemblyName>
     <TargetFramework>$(NetCoreAppCurrent)</TargetFramework>
+    <TargetFramework Condition="'$(BuildingInsideVisualStudio)' == 'true'">netcoreapp3.0</TargetFramework>
```

Comment:
OK, I think the right fix would be to downgrade this project entirely. So to set TargetFramework to netcoreapp3.0 not just for inside VS as it was before. We can upgrade at a late rpoint. Same for crossgen2.

Reply:
I actually think it was a good idea when you upgraded it to 5.0, since this is the shipping configuration.

The netcoreapp3.0 is just a temporary internal scenario used by our team, and we can remove it when dotnet5 starts working with VS.